katsdpcontroller/product_config.py

```diff
@@ -318,7 +358,7 @@ def __init__(self, name: str, src_streams: Sequence[Stream], *,
 
 
 class AntennaChannelisedVoltageStream(CbfStream, AntennaChannelisedVoltageStreamBase):
-    """Real antenna-channelised-voltage stream."""
+    """Real antenna-channelised-voltage stream (old correlator)."""
```

Comment:
I'm wondering about the naming.  "Next-generation" and "old" make sense now, but what happens some time in the future when we bring out the next next-generation correlator?

Reply:
This is the topic of https://skaafrica.atlassian.net/browse/NGC-199 :-) I'll update once we have a resolution there.

Reply:
Perfect 🙂